test/base.js

```diff
@@ -20,36 +20,36 @@ describe('base config', () => {
         unlinkConfig();
     });
 
-    it('should load properly base config', () => {
-        const cli = new eslint.CLIEngine({
+    it('should load properly base config', async () => {
+        const cli = new eslint.ESLint({
```

Comment:
For details on the ESLint API changes, see:

- https://eslint.org/docs/8.0.0/user-guide/migrating-to-8.0.0#remove-cliengine
- https://eslint.org/docs/developer-guide/nodejs-api#-eslintlinttextcode-options